man/kernel-install.xml

```diff
@@ -174,9 +171,44 @@
     <para><varname>KERNEL_INSTALL_MACHINE_ID=</varname> is set for the plugins to the desired <replaceable>MACHINE-ID</replaceable>.</para>
     <para><varname>KERNEL_INSTALL_BOOT_ROOT=</varname> is set for the plugins to the detected EFI system partition mount-point.</para>
 
-    <para><varname>KERNEL_INSTALL_BOOTLOADER_SPEC=0|1</varname> is set for the plugins to configure whether they should operate on the
-    <ulink url="https://systemd.io/BOOT_LOADER_SPECIFICATION">Boot Loader Specification</ulink> hierarchy.
-    This corresponds to the existence of <filename>$BOOT/<replaceable>MACHINE-ID</replaceable></filename>.</para>
+    <para><varname>KERNEL_INSTALL_FLAVOUR=legacy|BLS-EFI|BLS-unified|BLS-EBLP</varname>
```

Comment:
"flavour" is not good, because of the difference in spelling between imperial and colonial dialects. Maybe just call this
`KERNEL_INSTALL_LAYOUT` ?

Also, our internal options (on the right hand side) are always lowercase, to make it easy to write. So 'BLS-EFI' should be 'bls-efi', etc.

Actually, I don't think we need enumerate all the layouts here. `kernel-install` currently only supports "bls-efi" through `90-loaderentry.install`. So maybe just describe what "bls-efi" does, and say that other formats may be defined, but need to be handled through appropriate plugins.

We *could* add support for the "bls-unified" format. It'd require taking the initrd and concatenating it with the kernel image and some other stuff. But we can add it to the documentation here when support is added.

Reply:
Fair enough, updated